### PR TITLE
CI for PR Action

### PR DESCRIPTION
**Pull request by abassey2**
Build and Run a Docker container for the backend and client of RelayRunner.

Closes: https://github.com/retaildevcrews/wcnp/issues/209

### CODE REVIEW

File: .github/workflows/relayrunner-pr.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                sleep 1
+                ((wait_time=wait_time+1))
+            done
+
+    - name: Docker Tag and Push
+      run: |
+        # tag the repo with :dev
+        docker tag image $DOCKER_REPO:dev
+        # Push to the repo
+        docker push -a $DOCKER_REPO
```
Comment (bartr) [inline .github/workflows/relayrunner-pr.yml]: linter - EOL

File: .github/workflows/relayrunner-pr.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      - '**.css'
+      - '.github/workflows/relayrunner-pr.yaml'
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_REPO: abassey2/relayrunner
```
Comment (bartr) [inline .github/workflows/relayrunner-pr.yml]: should you push this to retaildevcrews repo?

Comment (abassey2) [reply .github/workflows/relayrunner-pr.yml]: There is no image on the retaildevcrews repo and I didn't have access to create the image. Could you create the image so I can build and push it to that repo?

File: .github/workflows/relayrunner-pr.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Login to Container Registry
+      uses: docker/login-action@v1
+      with:
```
Comment (bartr) [inline .github/workflows/relayrunner-pr.yml:26]: use this to push to retaildevcrews/relayrunner

registry: ghcr.io
username: ${{ secrets.GHCR_ID }}
password: ${{ secrets.GHCR_PAT }}

GHCR_* are "organization" secrets

File: .github/workflows/relayrunner-pr.yml (hunk 2 above)
Comment (bartr) [reply .github/workflows/relayrunner-pr.yml]: see comment below on how to do this

by default, the image will be private. you should be able to make it public - if not, email me and I can.

Comment (abassey2) [reply .github/workflows/relayrunner-pr.yml]: Where can I view the image and change it to public?

Comment (bartr) [reply .github/workflows/relayrunner-pr.yml]: https://github.com/orgs/retaildevcrews/packages

Comment (abassey2) [reply .github/workflows/relayrunner-pr.yml]: hmm the image is not popping up for me. I am not sure what happened

**Pull request merged**